AdminWebsite/AdminWebsite/ClientApp/src/app/work-allocation/edit-work-hours/vho-work-hours-non-availability-table/vho-work-hours-non-availability-table.component.ts

```diff
@@ -453,5 +455,10 @@ export class VhoWorkHoursNonAvailabilityTableComponent implements OnInit, CanDea
         } else if (this.nonWorkHours.length === 0) {
             this.showMessage(VhoWorkHoursNonAvailabilityTableComponent.WarningNoWorkingHoursForVho);
         }
+
+    }
+    public filterByFutureDate(value: VhoNonAvailabilityWorkHoursResponse[]) {
+        const todayDate = new Date();
```

Comment:
Make todays data a property of the component, then we can test this without updating the other unit tests

Reply:
I agree, we should make the date property flexible else the tests will fail next week